### DISCUSSION TITLE
Customized inputs file for Moving EB_CNS problem

### DISCUSSION THREAD

**Discussion by JerryNiOnly** [AMReX-Codes/amrex#3264] [Q&A] [answered]
Hi! I've been trying to use AMReX EB_CNS to reporduce the results exhibited in the paper "A moving embedded boundary approach for the compressible Navier-Stokes equations in a block-structured adaptive refinement framework" by Natarajan, Mahesh; Grout, Ray; Zhang, Weiqun; Day, Marc (https://ui.adsabs.harvard.edu/abs/2022JCoPh.46511315N/abstract)

I've been trying to edit the inputs files and prob_parm headfiles in ShockRef and Sod examples to create a moving geometry but failed. Is it possible to recurrent the numerial simualtions mentioned in the paper by simply editing input data or is it necessary to edit the core codes?

I found a post (https://github.com/AMReX-Codes/amrex/discussions/2914) regarding this problem. May I ask if I can make a query for the repo? Thank you very much!

**Comment by drummerdoc** [comment, marked as answer]
Yeah, that was in a completely separate repo that was branched off of an old version of AMReX as a one-off and then customized extensively.  None of that functionality was pulled back into AMReX directly.  Mahesh, a former member of my group, did the coding for that independently and I have a copy of the repo when he left our group.  Reach out to me at Marc.Day@nrel.gov privately and I'll help get you a copy of that to look at and play with.  Note that there is a move to reimplement those ideas in the context of a more modern AMReX release - perhaps @asalmgren can provide info for this more current work.

**Reply by JerryNiOnly** [reply]
Thank you very much!

**Comment by asalmgren** [comment]
We are working on an incompressible moving EB code based on AMReX but that is not ready to be open-source yet.  However, we don't currently have plans to provide the moving EB-CNS code within AMReX itself.
